### PR TITLE
Fix formatting of PR ping Slack messages

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Follow up #3577 and #3651 

## Description

Last notification of new PR received has bad formatting.

CleanShot 2024-01-11 at 14 50 49@2x

This is a quick PR to fix it, removing leftovers of when the action used curl calls.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — APPROVED** [review]

**Comment by sarayourfriend** [comment]
The PR limit reminders job is also failing. I'll look into it now.

File: .github/workflows/pr_ping.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         with:
           payload: |
             {
-             "text": "PR by \"${{ env.pr_author }}\" in \"${{ env.pr_repo }}\" marked ready for review: #${{ env.pr_number }} - \"${{ env.pr_title }}\"",
+             "text": "PR by${{ env.pr_author }}in ${{ env.pr_repo }} marked ready for review: #${{ env.pr_number }} - ${{ env.pr_title }}",
```
Comment (dhruvkb) [inline .github/workflows/pr_ping.yml:54]:
```suggestion
             "text": "PR by ${{ env.pr_author }} in ${{ env.pr_repo }} marked ready for review: #${{ env.pr_number }} - ${{ env.pr_title }}",
```

File: .github/workflows/pr_ping.yml [hunk 2]
```diff
@@ -33,13 +33,13 @@ jobs:
...
             {
-              "text": "New \"${{ env.pr_status }}\" PR opened by \"${{ env.pr_author }}\" in \"${{ env.pr_repo }}\": #\"${{ env.pr_number }}\" - \"${{ env.pr_title }}\"",
+              "text": "New${{ env.pr_status }}PR opened by ${{ env.pr_author }} in ${{ env.pr_repo }}: #${{ env.pr_number }} - ${{ env.pr_title }}",
               "blocks": [
                 {
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "\"${{ env.pr_icon }}\" New\"${{ env.pr_status }}\"PR opened by *\"${{ env.pr_author }}\"* in `\"${{ env.pr_repo }}\"`:\n<\"${{ env.pr_url }}\"|#\"${{ env.pr_number }}\" - \"${{ env.pr_title }}\">"
+                    "text": "${{ env.pr_icon }} New${{ env.pr_status }}PR opened by *${{ env.pr_author }}* in `${{ env.pr_repo }}`:\n<${{ env.pr_url }}|#${{ env.pr_number }} - ${{ env.pr_title }}>"
```
Comment (dhruvkb) [inline .github/workflows/pr_ping.yml:42]:
```suggestion
                    "text": "${{ env.pr_icon }} New ${{ env.pr_status }} PR opened by *${{ env.pr_author }}* in `${{ env.pr_repo }}`:\n<${{ env.pr_url }}|#${{ env.pr_number }} - ${{ env.pr_title }}>"
```

File: .github/workflows/pr_ping.yml [hunk 3]
```diff
@@ -33,13 +33,13 @@ jobs:
         with:
           payload: |
             {
-              "text": "New \"${{ env.pr_status }}\" PR opened by \"${{ env.pr_author }}\" in \"${{ env.pr_repo }}\": #\"${{ env.pr_number }}\" - \"${{ env.pr_title }}\"",
+              "text": "New${{ env.pr_status }}PR opened by ${{ env.pr_author }} in ${{ env.pr_repo }}: #${{ env.pr_number }} - ${{ env.pr_title }}",
```
Comment (dhruvkb) [inline .github/workflows/pr_ping.yml:36]:
```suggestion
              "text": "New ${{ env.pr_status }} PR opened by ${{ env.pr_author }} in ${{ env.pr_repo }}: #${{ env.pr_number }} - ${{ env.pr_title }}",
```

**Review by dhruvkb — APPROVED** [review]
Some spaces between the words are missing, except that it LGTM!

File: .github/workflows/pr_ping.yml (hunk 3 above)
Comment (krysal) [reply .github/workflows/pr_ping.yml:36]: The `env.pr_status` status includes the space 😃

File: .github/workflows/pr_ping.yml (hunk 2 above)
Comment (krysal) [reply .github/workflows/pr_ping.yml:42]: Same here.

File: .github/workflows/pr_ping.yml (hunk 1 above)
Comment (krysal) [reply .github/workflows/pr_ping.yml:54]: And this is indeed a mistake, well spotted. Thanks!

**Pull request merged**